mmengine/visualization/visualizer.py

```diff
@@ -399,6 +400,8 @@ def draw_texts(
             vertical_alignments: Union[str, List[str]] = 'top',
             horizontal_alignments: Union[str, List[str]] = 'left',
             font_families: Union[str, List[str]] = 'sans-serif',
+            font_properties: Optional[Union[FontProperties,
+                                            List[FontProperties]]] = None,
```

Comment:
Adding a new parameter in the middle position may cause a bc issue. Suggest moving it to the end.

Reply:
ok~